ironfish/src/testUtilities/fixtures.ts

```diff
@@ -45,7 +46,7 @@ export async function useFixture<TFixture, TSerialized = unknown>(
   } = {},
 ): Promise<TFixture> {
   const testPath = getCurrentTestPath()
-  const testName = expect.getState().currentTestName
+  const testName = expect.getState().currentTestName || ''
```

Comment:
I'm not sure under what condition these would be null, so it should probably just throw an error here so something unexpected doesn't happen.

Reply:
Seems like this was returning undefined or an empty string before, and we call this outside of some tests via `makeDbName`. Leaving it for now